src/Stan/Config.hs

```diff
@@ -207,10 +219,111 @@ applyChecks paths = foldl' useCheck (mkDefaultChecks paths)
     applyForScope f mScope hm = case mScope of
         Nothing -> f <$> hm  -- no scope = apply for everything
         Just cScope -> case cScope of
-            CheckScopeFile path -> HashMap.alter (fmap f) path hm
+            CheckScopeFile path -> HashMap.adjust f path hm
             CheckScopeDirectory dir -> HashMap.mapWithKey
                 (\path -> if isInDir dir path then f else id)
                 hm
 
     isInDir :: FilePath -> FilePath -> Bool
     isInDir dir path = dir `isPrefixOf` path
+
+{- $applyConfig
+
+The 'applyConfig' function transforms the list of rules defined in the
+'Config' (either via TOML or CLI) to get the list of 'Inspection's for
+each module.
+
+By default, @stan@ runs all 'Inspection's for all modules in the
```

Comment:
All defined in stan? Or will it be +custom? Or should we not worry about that?

Reply:
In the future it will be `+custom`. But I don't think we should worry about that in the moment, we can patch the docs when we implement custom inspections...